Marlin/pins_VOXEL8_GEN3B.h

```diff
@@ -36,60 +36,67 @@
+#define X_STEP_PIN              37  // Actuate X stepper driver
+#define X_DIR_PIN               48  // X Stepper driver direction
+#define X_MIN_PIN               24  // X min limit switch
+#define X_MAX_PIN               76  // X max limit switch
+#define X_ENABLE_PIN            29  // Enable X stepper driver
+#define X_MS1_PIN               40  // X stepper driver microstep select 1
+#define X_MS2_PIN               41  // X stepper driver microstep select 2
+
+#define Y_STEP_PIN              36  // Actuate Y stepper driver
+#define Y_DIR_PIN               49  // Y Stepper driver direction
+#define Y_MIN_PIN               23  // Y min limit switch
+#define Y_MAX_PIN               77  // Y max limit switch
+#define Y_ENABLE_PIN            28  // Enable Y stepper driver
+#define Y_MS1_PIN               69  // Y stepper driver microstep select 1
+#define Y_MS2_PIN               39  // Y stepper driver microstep select 2
+
+#define Z_STEP_PIN              35  // Actuate Z stepper driver
+#define Z_DIR_PIN               47  // Z Stepper driver direction
+#define Z_MIN_PIN               73  // Z min limit switch
+#define Z_MAX_PIN               30  // Z max limit switch
+#define Z_ENABLE_PIN            27  // Enable Z stepper driver
+#define Z_MS1_PIN               68  // Z stepper driver microstep select 1
+#define Z_MS2_PIN               67  // Z stepper driver microstep select 2
 
 /************************
     DIGITAL POTENTIOMETER PINS
 *************************/
 // Currently used for limiting motor current
 #undef DIGIPOTSS_PIN
-#define DIGIPOTSS_PIN 38              // Chip Select for digipot SPI communication
-#define DIGIPOT_CHANNELS {4,5,3,0,1}  // X Y Z E0 E1 digipot channels to stepper driver mapping
+#define DIGIPOTSS_PIN           38  // Chip Select for digipot SPI communication
+#define DIGIPOT_CHANNELS        {4,5,3,0,1}  // X Y Z E0 E1 digipot channels to stepper driver mapping
 
 /*************************
 	  HEATED BED PINS
 *************************/
-#define BED_AVAIL_PIN   82  // From key circuit
-#define HEATER_BED_PIN  4   // Drive the heated bed
-#define TEMP_BED_PIN    2   // Heated bed thermistor
+#define BED_AVAIL_PIN           82  // From key circuit
+#define HEATER_BED_PIN          4   // Drive the heated bed
+#define TEMP_BED_PIN            2   // Heated bed thermistor
 
 /*************************
 	  CARTRIDGE PINS
 *************************/
-#define CART1_SIG1_PIN  5   // PWM Capable
-#define CART1_SIG2_PIN  46  // PWM Capable
-#define CART1_SIG3_PIN  74  // 
-#define CART2_SIG1_PIN  45  // PWM Capable
-#define CART2_SIG2_PIN  75  // Interrupt Capable
-#define CART2_SIG3_PIN  85
+#define CART0_SIG1_PIN          5   // PWM Capable
+#define CART0_SIG2_PIN          46  // PWM Capable
+#define CART0_SIG3_PIN          74  // 
+#define CART1_SIG1_PIN          45  // PWM Capable
+#define CART1_SIG2_PIN          75  // Interrupt Capable
+#define CART1_SIG3_PIN          85
 
```

Comment:
We are indexing the cartridges from 0. Should we not also index the signals from 0?